speccurses.py

```diff
@@ -492,7 +506,7 @@ def _handle(msg):
     for r in config.runs:
         # draw_status_bar(stdscr, "Starting run '{}'".format(r.tag))
 
-        result = r.run(path, _handle)
+        result = r.run(path, _handle, lambda x:x)
```

Comment:
If we're passing an additional argument to handle errors, shouldn't we pass something other than this? Should this be the default argument for `run()`?

Reply:
Both stdout and stderr are being written to log files already, so it’s not crucial the text be shown in console as well.  Specjbb uses stderr for a bunch of status messages and clutters things up I think.